engineering/conventions-go.md

```diff
@@ -62,4 +57,10 @@ A blank line is usually added after the end of any control structure, too.
 
 ### Naming
 
-* Prefer full names (e.g. `Repository`, not `Repo`).
\ No newline at end of file
+* Prefer full names (e.g. `Repository`, not `Repo`).
+
+## CLI
+
+* Put sources for your executable commands in `cmd/<command-name>/main.go`.
+* We use [go-flags](https://github.com/jessevdk/go-flags) extensively for CLI options parsing.
+* Implement commands in a subpackage ([example](https://github.com/src-d/gitbase/blob/master/cli/gitbase/cmd/server.go)).
```

Comment:
bad example, it is using `cli` (there is a PR fixing it)

Reply:
@ajnavarro Ok. Once the PR is merged I'll update the link. Or maybe there is a better example?